### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by zsau** [tombenner/NUIParse#4]
Is NUIParse still tracking CoreParse? If not, it might be a good idea to copy over the documentation, so that any incompatible changes to CoreParse don't leave NUIParse with misleading docs.

**Comment by timbodeit** [comment]
Which documentation are you talking about? The rest of the Readme?
The only other documentation of CoreParse I am aware of is the inline documentation.

**Comment by zsau** [comment]
Yes, sorry, the README. It would also be nice to know which project to submit bug reports and feature requests to.
